tests/mock_s3_server/mock_s3_server.py

```diff
@@ -436,7 +435,7 @@ def handle_get_object(wrapper, request, parsed_path, head_request=False):
 
     if (parsed_path.path == "/get_object_invalid_response_missing_content_range" or
         parsed_path.path == "/get_object_invalid_response_missing_etags" or
-        parsed_path.path == "/get_object_long_error"):
+            parsed_path.path == "/get_object_long_error"):
```

Comment:
This tab seems weird. We don't have it for the above line.

Reply:
I used
```
(venv) dengket@80a99735f9a4 test_helper % python -m autopep8 --version                 
autopep8 2.3.1 (pycodestyle: 2.12.1)
```

That added the tap